lib/ueberauth/strategy/amco.ex

```diff
@@ -182,8 +182,8 @@ defmodule Ueberauth.Strategy.Amco do
     %Info{
       email: user["email"],
       phone: user["phone_number"],
-      last_name: user["last_name"],
-      first_name: user["first_name"]
+      last_name: user["family_name"],
```

Comment:
I see you are changing everything to family_name and Given_name, shouldn't these keys also change?

Reply:
The standard for the data/claims the idp should return are given_name and family_name. But the current user struct is gonna use this fields as first_name and last_name.